lib/routes/api/v1/favorites.js

```diff
@@ -2,7 +2,8 @@ const express = require('express');
 const router  = express.Router();
 const favoritesController = require('../../../controllers/favorites_controller')
 
+router.get('/:id', favoritesController.show);
```

Comment:
I'm not sure if ordering matters here like it does in Rails, but it is probably safer to put it below the other get requests here.

Reply:
I re ordered them